scripts/build/build/__init__.py

```diff
@@ -139,11 +139,15 @@ def CleanOutputDirectories(self):
     # any generated output was cleaned
     self.completed_steps.discard(BuildSteps.GENERATED)
 
-  def CopyArtifactsTo(self, path: str):
+  def CopyArtifactsTo(self, path: str, compress: bool = False):
```

Comment:
nit:
```py
builder.CopyArtifactsTo('/foo/bar', True) 
```

is not immediately obvious of what 'true/false' actually means. Consider splitting into 'CompressArtifactsTo' or making the argument an enum, so you have `CopyArtifactsTo(path, Formats.COMPRESSED)` for better future API clarity.